### ISSUE TITLE
About Prediction task

### ISSUE THREAD

**Issue by Ycy21253728** [WenjieDu/PyPOTS#238]
### Issue description

Dear author,I noticed the need to have category labels in the training set. However, I want to do a prediction task, no category label, can I still use your network? If possible, please tell me what to do.

**Comment by WenjieDu** [comment]
Hi there 👋,

Thank you so much for your attention to PyPOTS! You can follow me on GitHub to receive the latest news of PyPOTS. If you find PyPOTS helpful to your work, please star⭐️ this repository. Your star is your recognition, which can help more people notice PyPOTS and grow PyPOTS community. It matters and is definitely a kind of contribution to the community.

I have received your message and will respond ASAP. Thank you for your patience! 😃

Best,
Wenjie

**Comment by WenjieDu** [comment]
Do you mean performing clustering on your dataset?

**Comment by Ycy21253728** [comment]
I want to imputation on my data set. But my data doesn't have categories.

**Comment by WenjieDu** [comment]
This shouldn't be a problem. Classification labels are not necessary for imputation models in PyPOTS. Maybe you should refer to the paper SAITS https://arxiv.org/abs/2202.08516 for more details and to learn more about time series imputation.

**Comment by Ycy21253728** [comment]
Sorry, I only looked at that simple example at first. Thank you for your reply.

**Comment by WenjieDu** [comment]
No problem. Let me know if you have further questions. And you are welcome to join our community https://github.com/WenjieDu/PyPOTS/#-community to help build PyPOTS better together ;-)
